predicate.csv

```diff
@@ -0,0 +1,6 @@
+predicate,object_type
+rdfs:label,Literal
+owl:sameAs,URIRef
+rdfs:subClassOf, URIRef
```

Comment:
There are extraneous leading spaces in the object_type column of these last three lines, which caused the `load_predicates` command to incorrectly assume a predicate that was already in the database was not.

To fix, remove the extra spaces from this file. It might also be a good idea to update the `load_predicates` command to run `.strip()` on the raw values it gets from the CSV file, in order to guard against this sort of accidental extra spaces in the future.